_maps/shrike.json

```
@@ -10,6 +10,19 @@
 		"ferry": "ferry_kilo",
 	    "emergency": "emergency_void"},
     "mine_disable": 1,
-	"ship_type": "/obj/structure/overmap/nanotrasen/missile_cruiser/starter/shrike",
-	"starmap_path": "config/starmap/starmap.json"
+	"ship_type": "/obj/structure/overmap/nanotrasen/frigate/starter/shrike",
+	"starmap_path": "config/starmap/starmap.json",
+
+	"map_description": "The Shrike was once in service some time ago but has recently been unearthed as a schematic for cheap, fast ships perfect for use as a QRF. Most if not all Shrike-Class ships have been abandoned for some time in one shipyard or another until Whiterapids bought them en masse and outfitted them to the bare minimum requirements. As a result, Shrike vessels lack some amenities and technologies, though they leave plenty of room for crew improvements.",
+	"manufacturer": "FrozenStar Armory",
+	"mapper": "Karmic-Skink",
+	"commission_date": "May 23, 2261",
```

```suggestion
	"commission_date": "May 23, 2175",
```
Re: discussion regarding this feature